presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestQueues.java

```diff
@@ -51,180 +52,170 @@
 {
     // Copy of TestQueues with tests for db reconfiguration of resource groups
     private static final String LONG_LASTING_QUERY = "SELECT COUNT(*) FROM lineitem";
+    private DistributedQueryRunner queryRunner;
+    private H2ResourceGroupsDao dao;
+
+    @BeforeMethod
+    public void setup()
+            throws Exception
+    {
+        String dbConfigUrl = getDbConfigUrl();
+        dao = getDao(dbConfigUrl);
+        queryRunner = createQueryRunner(dbConfigUrl, dao);
+    }
+
+    @AfterMethod(alwaysRun = true)
+    public void tearDown()
+    {
+        QueryManager queryManager = queryRunner.getCoordinator().getQueryManager();
+        queryManager.getAllQueryInfo().forEach(queryInfo -> queryManager.cancelQuery(queryInfo.getQueryId()));
+        queryRunner.close();
+    }
 
     @Test(timeOut = 60_000)
     public void testRunningQuery()
             throws Exception
     {
-        try (DistributedQueryRunner queryRunner = getSimpleQueryRunner()) {
```

Comment:
Yes, I guess the reason that `createQueryRunner()` is used in other tests is to get a reference to the `dao`.